coalib/bearlib/languages/Language.py

```diff
@@ -302,6 +302,15 @@ class Language(metaclass=LanguageMeta):
     >>> (Language.TrumpScript == 3.6) | (Language.TrumpScript == 2)
```

Comment:
It's better to make that also not work anymore by removing special comparison methods from `LanguageMeta` and also implementing `__and__` and `__or__` there